content/en/tracing/guide/add_span_md_and_graph_it.md

```diff
@@ -0,0 +1,278 @@
+If you want to add span metadata using a tag attached to a `Context`, call the `SpanFromContext` function:
+
+```go
+package main
+
+import (
+    "net/http"
+
+    "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+func handler(w http.ResponseWriter, r *http.Request) {
+    // Retrieve a span for a web request attached to a Go Context.
+    if span, ok := tracer.SpanFromContext(r.Context()); ok {
+        // Set tag
+        span.SetTag("customer_id", customer.id)
+    }
+}
+```
+
+{{% /tab %}}
+{{% tab "Node.js" %}}
+
+
+Add metadata directly to span objects by calling `setTag` or `addTags`:
+
+```javascript
+// An example of an Express endpoint,
+// with Datadog tracing around the request.
+app.get('/posts', (req, res) => {
+  const span = tracer.startSpan('web.request')
+
+  span.addTags({
+    'customer_id': customer.id
+  })
+})
+```
+
+Access the current active span from any method within your code.
+**Note**: If the method is called and there is no span currently active, `tracer.scope().active()` returns `null`.
+
+```javascript
+// e.g. adding tag to active span
+
+const span = tracer.scope().active()
+
+span.setTag('customer_id', customer.id)
+```
+
+{{% /tab %}}
+{{% tab ".NET" %}}
+
+Add metadata directly to a `Datadog.Trace.Span` object by calling `Span.SetTag()`. For example:
+
+```csharp
+using Datadog.Trace;
+
+// access the active scope through the global tracer (can return null)
+var scope = Tracer.Instance.ActiveScope;
+
+// add a tag to the span
+scope.Span.SetTag("customer_id", customer.id);
+```
+
+**Note**: `Datadog.Trace.Tracer.Instance.ActiveScope` returns `null` if there is no active span.
+
+{{% /tab %}}
+{{% tab "PHP" %}}
+
+
+Add metadata directly to a `DDTrace\Span` object by calling `Span::setTag()`. For example:
+
+```php
+// Get the currently active span (can be null)
+$span = \DDTrace\GlobalTracer::get()->getActiveSpan();
+if (null !== $span) {
+  // Add a tag to the span
+  $span->setTag('customer_id', customer.id);
+}
+```
+
+**Note**: `Tracer::getActiveSpan()` returns `null` if there is no active span.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+<div class="alert alert-info">You might have to wait a few minutes between deploying your updated code and seeing the new metadata in the Datadog UI</div>
+
+## Leverage the Datadog UI to search for your custom metadata
+
+2) **Go to the Services page** and click on the service that you added metadata to. **Scroll down and click on the specific resource** where the metadata was added in the Resource table.
+
+{{< img src="tracing/guide/add_span_md_and_graph_it/span_md_2.png" alt="comparaison 2" responsive="true" style="width:90%;">}}
+
+The span summary table allows to quickly identify any particular units within the operations that impact the resource. For example, here you can easily find the spans that take the largest portion of average operation time and investigate if they can be optimized.
+
+3) **Scroll down to the Traces table** 
```

Comment:
Similarly - this is turning into a walkthrough of the resource page and not the benefits of adding trace metadata. I think you can scrap this as well in favor of dashboarding/monitoring on metadata

Reply:
Can you give a good example for a dashboard widget I can easily build in the demo app with customer_id and a trace.* metric? I looked for good public about this and couldn't find any (unless you mean export from the App Analytics screen?).

Reply:
Spoke with Andrew about this IRL, made a last adjustment to resolve.